lte/gateway/python/integ_tests/s1aptests/test_attach_detach_secondary_pdn_disconnect_dedicated_bearer.py

```diff
@@ -20,7 +20,7 @@
 from integ_tests.s1aptests.s1ap_utils import SpgwUtil
 
 
-class TestSecondaryPdnDisConnDedBearerReq(unittest.TestCase):
+class TestAttachDetachSecondaryPdnDisconnectDedicatedBearer(unittest.TestCase):
     """Test secondary pdn connection and disconnection"""
```

Comment:
Could you correct the comment as it is not aligned with test case name

Reply:
Now we describe the testcase behavior in testcase function only and don't duplicate with the class name. For the class name comment we only mention the Integration test and the class name. I have updated the comment